src/time.rs

```diff
@@ -7,23 +7,28 @@ use crate::text::to_base_string;
 
 pub fn timestamp() -> Result<String, CuidError> {
     SystemTime::now().duration_since(UNIX_EPOCH)
-        .map(|time| time.as_secs())
+        .map(|time| time.as_millis())
         .map(to_base_string)
         .unwrap_or(Err(CuidError::TextError("Could not convert time to str")))
 }
 
 
-#[cfg(tests)]
+#[cfg(test)]
```

Comment:
Due to typo, this test section had never run, so I had to fix `test_timestamp` as well.